src/kernel_config_manager.sh

```diff
@@ -140,6 +140,7 @@ function save_config_file()
   if [[ "$ret" == 1 ]]; then
     warning "Warning: $name: there's nothing new in this file"
   elif [[ "$ret" -gt 0 ]]; then
+    git reset --hard --quiet # Aborts save operation
```

Comment:
Given #723 I don't think that this commit is necessary.

Reply:
Thanks for the clarification @kwy95, lesson learned 👍
Also think the commit is outdated with #723